src/main/java/org/dice_research/opal/civet/metrics/ProviderIdentityMetric.java

```diff
@@ -153,30 +153,30 @@ public Integer compute(Model model, String datasetUri) throws Exception {
 					Resource distribution_resource = (Resource) distribution;
 					if (distribution_resource.hasProperty(DCAT.accessURL)
```

Comment:
Nice name conventions, there is just a single name left, maybe you forgot.
please change distribution_resoure to distributionResource
line no: 153,154,155

Reply:
Thanks for catching this. It is done.